### ISSUE TITLE
diamond all-versus-all - No such file or directory

### ISSUE THREAD

**Issue by thackl** [davidemms/OrthoFinder#89]
Hi David,

over the last days I've been running orthofinder for several groups of interest. Mostly it worked great! However, for one group, the largest in my set with 666 genomes of the cyanobacterium Prochlorococcus, (also the group I am most  interested in), I am running into the following problem.  When starting diamond all-vs-all, error messages about missing file start to appear:

```
...
2017-06-21 10:54:21 : Creating diamond database 665 of 666
2017-06-21 10:54:21 : Creating diamond database 666 of 666

Running diamond all-versus-all
------------------------------
Using 20 thread(s)
2017-06-21 10:54:27 : This may take some time....
No such file or directory
Error: Error calling unlink.
No such file or directory
Error: Error calling unlink.
No such file or directory
Error: Error calling unlink.
No such file or directory
Error: Error calling unlink.
...
```

Which than ultimately results in the following error:

```
Running OrthoFinder algorithm
-----------------------------
2017-06-22 02:23:00 : Initial processing of each species
Malformatted line in /nobackup1b/users/chisholmlab/simons_metagenomes/mg-quantification/orthofinder/pro-v2/Results_Jun21/WorkingDirectory/Blast3_269.txt
Offending line was:

2017-06-22 02:24:06 : Initial processing of species 0 complete
2017-06-22 02:25:04 : Initial processing of species 1 complete
2017-06-22 02:25:52 : Initial processing of species 2 complete
ERROR: An error occurred, please review previous error messages for more information.
Done Thu Jun 22 02:26:20 EDT 2017
(END) 
```

Any idea what could be going on?

P.S.: I've started to run the diamond jobs manually one by one to save logs etc. Will let you know if something useful comes up

**Comment by davidemms** [comment]
Hi

I wonder if perhaps you're running out of space or you're creating more files than your system can handle? With 666 genomes you would get 443,556 diamond results files. Depending on the size of the genomes each file could take up anywhere from 50K to 200M so size (and time) are possible constraints. Do you know how many of the diamond results files were done before the error occurred?

All the best
David

**Comment by davidemms** [comment]
Hi

I've started to see this message too on a large analysis I'm running, did you manage to find anything out about it?

Thanks!
David

**Comment by davidemms** [comment]
I've reported this on the DIAMOND github page: https://github.com/bbuchfink/diamond/issues/175

**Comment by davidemms** [comment]
This is now fixed with the latest version of diamond, v0.9.11.
